src/kerberos_common.h

```diff
@@ -26,19 +26,33 @@ inline void ResultDeleter(krb_result* result) {
 NAN_INLINE std::string StringOptionValue(v8::Local<v8::Object> options, const char* _key) {
     Nan::HandleScope scope;
     v8::Local<v8::String> key = Nan::New(_key).ToLocalChecked();
-    return !options.IsEmpty() && options->Has(key) && options->Get(key)->IsString()
-               ? std::string(*(Nan::Utf8String(options->Get(key))))
-               : std::string();
+    if (options.IsEmpty() || !Nan::Has(options, key).FromJust()) {
```

Comment:
Wouldn't it be safer to do use `FromMaybe(false)` here? The implication of the [`Nan::Maybe` docs](https://github.com/nodejs/nan/blob/master/doc/maybe_types.md#nanmaybe) is that `FromJust` has the potential to crash.

Reply:
:+1: